client/my-sites/stats/activity-log-banner/activity-log-banner.jsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Gridicon from 'gridicons';
+
+class ActivityLogBanner extends Component {
+	static propTypes = {
+		icon: PropTypes.string,
+		isDismissable: PropTypes.bool.isRequired,
+		onDismissClick: PropTypes.func,
+		status: PropTypes.oneOf( [
+			'error',
+			'info',
+			'success',
+			'warning',
+		] ),
+		title: PropTypes.node.isRequired,
+		translate: PropTypes.func.isRequired,
+	};
+
+	static defaultProps = {
+		isDismissable: false,
+		onDismissClick: noop,
+		status: null,
+		title: '',
+	};
+
+	getIcon() {
+		// Allow an icon prop to override auto-icon (even null!)
+		const { icon } = this.props;
+		if ( typeof icon !== 'undefined' ) {
+			return icon;
+		}
+
+		switch ( this.props.status ) {
+			case 'error':
+			case 'warning':
+				return 'notice-outline';
+
+			case 'info':
+				return 'info-outline';
+
+			case 'success':
+				return 'star';
+		}
+
+		return null;
+	}
+
+	render() {
+		const {
+			isDismissable,
+			onDismissClick,
+			children,
+			title,
+			translate,
+			status,
+		} = this.props;
+
+		const icon = this.getIcon();
+		const classes = classNames( 'activity-log-banner', {
+			'is-error': 'error' === status,
```

Comment:
You don't need to specify any of these classes - if you use `highlight`, these classes will be added automatically to the `Card`.